src/components/VBtn/VBtn.ts

```diff
@@ -85,6 +85,11 @@ const VBtn = mixins(
       return (!this.outline && !this.flat)
         ? this.addBackgroundColorClassChecks(classes)
         : this.addTextColorClassChecks(classes)
+    },
+    computedRipple (): any {
```

Comment:
This returns `RippleOptions | boolean`, try to avoid using `any` where possible.